force-app/main/default/classes/PS_IntegrationServiceConfig.cls

```diff
@@ -78,11 +78,11 @@ public with sharing class PS_IntegrationServiceConfig {
     /**
     * @description Determines if the Elevate integration service is configured and enabled.
     * The Elevate Payment Services integration is considered enabled when all required
-    * service records are present with value and isActive record is either missing or true
+    * service records are present with value and Integration is Active
     * @return Boolean
     */
     public Boolean isIntegrationEnabled() {
-        return isIntegrationActive() && hasElevateServiceData();
+        return isIntegrationActive() && ElevateConfigService.hasAllRequiredServiceData();
```

Comment:
What do you think of also moving `hasAllRequiredServiceData()` to the Service class and directly call here. I can have the same pattern as `isIntegrationActive()` and call the ConfigService in the inner function. However, I don't see the value for `hasAllRequiredServiceData()` since the function name is pretty self explanatory in this context

Reply:
I'd say the only value at that point would be consistency. I'd say either we should go ahead and follow the same pattern as `isIntegrationActive` and call the service in the inner function or call the service directly here.